Bottom: 5d8a805bb77b155cef6b7c249c4420df5085cd85
Top:    780a23b7bf3dc726f266d981f01381e60f6c4224
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-20 11:39:37 +0800

Refresh of vhost_net-use-vhost_net

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 2cea3f1..23067c7 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -660,7 +660,7 @@ static void handle_tx_copy(struct vhost_net *net)
 		vq->heads[nheads].len = 0;
 
 		total_len += len;
-		err = vhost_net_build_xdp(nvq, &msg.msg_iter,
+		err = vhost_net_build_xdp(net, nvq, &msg.msg_iter,
 					  &nvq->xdp[nheads]);
 		if (!err) {
 			if (++nheads == VHOST_RX_BATCH) {
